ipywidgets/static/widgets/js/layout.js

```diff
@@ -36,18 +36,15 @@ define([
                 'align_items',
                 'align_self',
                 'bottom',
+                'border',
```

Comment:
That's like arguing font size is layout because it can affect the width and height of the object depending on the contents.  I would not add this, it's primary function is to set border style.  Nor would I add border-width-top/bottom/left/right.  All of that stuff should be set by CSS and not accessible in `Layout`